scripts/cubic/bootstrap-fmriprep.sh

```diff
@@ -218,41 +209,43 @@ chmod +x code/participant_job.sh
 cat > code/fmriprep_zip.sh << "EOT"
 #!/bin/bash
 set -e -u -x
-
 subid="$1"
 mkdir -p ${PWD}/.git/tmp/wdir
+# TODO: fix path to singularity image
 singularity run --cleanenv -B ${PWD} \
-    pennlinc-containers/.datalad/environments/fmriprep-20-2-3/image \
+    containers/images/bids/bids-fmriprep--20.2.3.sing \
     inputs/data \
     prep \
     participant \
-    -w ${PWD}/.git/tmp/wkdir \
+    -w ${PWD}/.git/wkdir \
```

Comment:
any specific reason to change this?

Reply:
again, I didn't change it, that was in the code I've got, so no reason behind